### PR TITLE
Pick a random person as the deployer and display their name

### PR DESCRIPTION
**Pull request by duncanjbrown**
- include a couple of understudies in case they're ill or away
- randomise a list of names from ENV
- make sure it returns the same random order over the course of a day

Screenshot 2020-01-13 at 09 44 35

### CODE REVIEW

File: spec/lib/deployers_spec.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    ClimateControl.modify DEPLOYERS: 'one,two,three,four,five,six,seven' do
+      ex.run
+    end
+  end
+
+  describe '.for_today' do
+    it 'always returns the same value on the same day' do
+      Timecop.travel(2019,1,1) do
+        deployers_for_today = Array.new(10) { Deployers.for_today }
+        expect(deployers_for_today.uniq.count).to eq 1
```
Comment (tvararu) [inline spec/lib/deployers_spec.rb:17]: I'm not sure I follow what is being asserted here; so it's checking if `["five", "six", "four", "two", "seven", "three", "one"].uniq.count` is 1?... That seems wrong...

**Review by tvararu — APPROVED** [review]
Just wondering about the test but :shipit:

File: spec/lib/deployers_spec.rb (hunk 1 above)
Comment (duncanjbrown) [reply spec/lib/deployers_spec.rb:17]: `Array.new(10) { Deployers.for_today }` will return an array of 10 elements each populated by a fresh call to `Deployers.for_today`, so it should be true that this array contains only one unique element

Comment (tvararu) [reply spec/lib/deployers_spec.rb:17]: Ah yes!

**Pull request merged**